framework/helpers/BaseArrayHelper.php

```diff
@@ -831,12 +831,13 @@ public static function isIndexed($array, $consecutive = false)
     }
 
     /**
-     * Check whether an array or [[Traversable]] contains an element.
+     * Check whether an array values or [[Traversable]] properties contains given value.
```

Comment:
I think description was alright and not it is not. We check that array or traversable contains an element specified.

Reply:
@samdark ok, roll back